src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1084,6 +1100,45 @@ public void sendBattleRecordsToGameData(final IDelegateBridge aBridge) {
     }
   }
 
+
+  /**
+   * 'Auto-fight' all of the air battles and strategic bombing runs.
+   * Auto fight means we automatically begin the fight without user action. This is to avoid clicks during the
+   * air battle and SBR phase, and to enforce game rules that these phases are fought first before any other combat.
+   */
+  void fightAirCombats(final IDelegateBridge delegateBridge) {
```

Comment:
So the problem with this name is there is also a battle type of AIR_BATTLE which you aren't fighting here. There are 3 different types of air related battle types: AIR_BATTLE, AIR_RAID, BOMBING_RAID. You are only fighting battles related to bombing here so I would suggest 'fightBombingBattles' or 'fightBombingRaids' or 'fightAirRaidsAndBombingRaids'. Personally I don't really like the battle type names we have but that is a bigger issue. Mostly just want to keep things consistent with the terms we already have.